infra/concourse/Makefile.BUILD

```diff
@@ -13,13 +13,13 @@ BUILD_RUBY_VERSION := 2.6.3
 # Adding a component or upgrading a component to a backwards compatible release should constitute a minor release.
 # Fixing bugs or making trivial changes should be considered a patch release.
 
-DOCKER_TAG_VERSION_TERRAFORM := 2.0.0
+DOCKER_TAG_VERSION_TERRAFORM := 2.1.0
 DOCKER_TAG_VERSION_KITCHEN_TERRAFORM := 2.1.0
 
 REGISTRY_URL := gcr.io/cloud-foundation-cicd
 
 DOCKER_IMAGE_LINT := cft/lint
-DOCKER_TAG_LINT := 2.3.0
+DOCKER_TAG_LINT := 3.0.0
```

Comment:
If we don't remove `terraform_docs.sh`, this can be `2.4.0`.

Reply:
Fixed